packages/cozy-stack-client/src/KonnectorCollection.js

```diff
@@ -17,8 +24,73 @@ class KonnectorCollection extends AppCollection {
     throw new Error('destroy() method is not available for konnectors')
   }
 
-  async update() {
-    throw new Error('update() method is not available for konnectors')
+  async findTriggers(slug) {
+    const triggerCol = new TriggerCollection(this.stackClient)
+    const { data: rawTriggers } = await triggerCol.all()
```

Comment:
all() is paginated no? So we are trying to `find triggers by slug in the last 100 triggers` ?

Reply:
Thx, I added `{ limit: null }`